root/io/issue-9899/read_cmssw_class_v3.cxx

```diff
@@ -43,11 +43,11 @@ struct Values {
 
 #ifdef __ROOTCLING__
 #pragma read sourceClass="RefVectorBase" targetClass="RefVectorBase" source="" \
-        version="1-" target="fTransient" code="{ fTransient.clear(); }"
+        version="[1-]" target="fTransient" code="{ fTransient.clear(); }"
```

Comment:
Looks to me like it's dash separate list, not comma separated list as the commit says. Also, while it is a list it only has one element. Should there be a test with more than one version in this list to actually make use of the feature? Maybe it's for another PR. Also the changes of this commit do not seem related to the title of the PR, could you elaborate there?

Reply:
> Looks to me like it's dash separate list, not comma separated list as the commit says.

Wherever I copy/pasted from has an incomplete description of the content.  What is supported is **both** a comma separated list of versions **and** a range (`[-X]` or `[X-]`).

The relevant part of the commit was actually the "enclosed in square brackets" part.  I will update the commit log.